modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/tracing/opentelemetry/management/LogTelemetryManager.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.aspects.flow.statistics.tracing.opentelemetry.management;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.extension.trace.propagation.B3Propagator;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.aspects.flow.statistics.tracing.opentelemetry.management.handling.span.OpenTelemetrySpanHandler;
+import org.apache.synapse.aspects.flow.statistics.tracing.opentelemetry.management.handling.span.SpanHandler;
+import org.apache.synapse.aspects.flow.statistics.tracing.opentelemetry.management.scoping.TracingScopeManager;
+
+public class LogTelemetryManager implements OpenTelemetryManager {
+
+    private static final Log logger = LogFactory.getLog(LogTelemetryManager.class);
+    private SdkTracerProvider sdkTracerProvider;
+    private OpenTelemetry openTelemetry;
+    private TelemetryTracer tracer;
+    private SpanHandler handler;
+
+    @Override
+    public void init() {
+        LogExporter logExporter = LogExporter.create();
+
+        if (logger.isDebugEnabled()) {
+            logger.debug("Log exporter: " + logExporter + " is configured");
+        }
+
+        Resource serviceNameResource = Resource.create(Attributes.of(ResourceAttributes.SERVICE_NAME,
+                TelemetryConstants.SERVICE_NAME));
+
+        sdkTracerProvider = SdkTracerProvider.builder()
+                .addSpanProcessor(BatchSpanProcessor.builder(logExporter).build())
+                .setResource(Resource.getDefault().merge(serviceNameResource))
+                .build();
+
+        openTelemetry = OpenTelemetrySdk.builder()
+                .setTracerProvider(sdkTracerProvider)
+                .setPropagators(ContextPropagators.create(B3Propagator.injectingMultiHeaders()))
```

Comment:
Any particular reason for using the `B3Propagator` specifically @Rashmika998 ?

Reply:
No reason. It works for JaegerPropagator as well.